R/target_market_share.R

```diff
@@ -482,3 +455,57 @@ check_valid_columns <- function(data, valid_columns) {
 
   invisible(data)
 }
+
+pivot_wider2 <- function(data, ...) {
+  abort_if_has_list_colums(data)
+
+  out <- suppressWarnings(pivot_wider(data, ...))
```

Comment:
out of curiosity, what warnings are you expecting?

Reply:
I actually have no idea. @maurolepore added this `pivot_wider2` to solve a problem we ran into ages ago, and I truly don't remember what this problem was...